frontAndBackExample/routes/users.js

```diff
@@ -3,7 +3,7 @@ const router = express.Router();
 
 //for passing stuff to and from the database
 require("dotenv/config");
-const newUserRegistration = require('../models/user');
+const {User} = require('../models/models');
```

Comment:
Any particular reason why the change of variable name?

Reply:
In terms of newUserRegistration vs User, you are directly accessing the User Model in this code. I wanted it to by obvious that you were accessing the User model, and by keeping the naming consistent with all the other times you use the User model, it becomes more clear. Just a small readability thing.